tests/integration.spec.coffee

```diff
@@ -149,13 +151,15 @@ describe 'SDK Integration Tests', ->
 
 				it 'should be rejected with an error', ->
 					promise = resin.auth.getEmail()
-					m.chai.expect(promise).to.be.rejectedWith(errors.ResinNotLoggedIn)
+					m.chai.expect(promise).to.be.rejected
+						.and.eventually.have.property('code', 'ResinNotLoggedIn')
```

Comment:
I think I'd rather use `code`, since it feels like a more explicit part of the interface than the name of constructor that was used, and I think it's probably what an end user would use. Happy to change it if you feel strongly though.